src/libs/actions/IOU.js

```diff
@@ -1290,10 +1255,10 @@ function splitBill(participants, currentUserLogin, currentUserAccountID, amount,
  * @param {String} comment
  * @param {String} currency
  * @param {String} category
+ * @param {String} tag
```

Comment:
Coming from https://github.com/Expensify/App/issues/34988
When creating the optimistic split transaction, we needed to pass the category & tag to `buildOptimisticTransaction`